projects.md

```diff
@@ -1,3 +1,27 @@
+---
+layout: contribute
+title: Mentoring Projects
+---
+
+## Mentoring Projects
+
+This page details potential projects and their mentors, for participants of
+internship programs such as [Google Summer of Code](https://www.google-melange.com/),
+[Outreachy](https://www.gnome.org/outreachy/), and so forth. Each program will
+have it's own requirements and application process - this page simply lists the
+currently available projects.
+
+## I'm interested - how can I get familiar with Foreman?
+
+* Grab a cup of your favorite beverage
+* Skim through the [manual](/documentation.html)
+* Install Foreman on a virtual machine or your own computer
```

Comment:
do we have a generic link? the quickstart "tab" at the top goes to 1.9, so that will need updating each release...